src/unit/test_bitops.c

```diff
@@ -6,13 +6,12 @@
 #include "../config.h"
 #include "../zmalloc.h"
 
+extern long long serverPopcount(void *s, long count);
 extern long long popcountScalar(void *s, long count);
 #if HAVE_X86_SIMD
+extern long long popcountAVX512(void *s, long count);
 extern long long popcountAVX2(void *s, long count);
 #endif
-#if defined(__aarch64__)
-extern long long popcountNEON(void *s, long count);
-#endif
```

Comment:
There is a test for popcountNEON guarded by a similar ifdef. If we remove the prototype, does it compile on ARM?

I don't know why the CI tests didn't run on this PR.

Reply:
I found the CI did run on your fork. The Mac build failed when this prototype was removed:

```
unit/test_bitops.c:55:30: error: call to undeclared function 'popcountNEON'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
        long long ret_neon = popcountNEON(buf, size);
                             ^
```

https://github.com/shanwan1/valkey/actions/runs/15786879582/job/44505112668#step:3:242